src/analyzer_status_reporter.ts

```diff
@@ -4,15 +4,15 @@ import { window, StatusBarItem, Disposable } from "vscode";
 import { Analyzer } from "./analyzer";
 import { ServerStatusNotification } from "./analysis_server_types";
 
-let statusBarItem: StatusBarItem;
-let statusShowing: boolean;
-
 export class AnalyzerStatusReporter extends Disposable {
+	statusBarItem: StatusBarItem;
```

Comment:
I think I should've actually made these private (I'm used to C#, things default to private!) so I'll try changing that later.